core/variant/variant_parser.cpp

```diff
@@ -90,6 +90,17 @@ const char *VariantParser::tk_name[TK_MAX] = {
 	"ERROR"
 };
 
+static double sfixtor(const String &p_sfix) {
```

Comment:
What do `sfixtor` and `sfix` mean?

Reply:
I guess it's a misunderstanding of what `rtosfix` means, trying to reverse this around?

`rtosfix` means "Real TO String FIX", i.e. it fixes the conversion of reals to strings.
So this should be `storfix` with e.g. parameter `p_str`.

Reply:
This code is carryover from the abandoned PR. I think `rtosfix` could use at at _minimum_ a brief comment, but I'm in favor of renaming it, say, `double_to_string_fix`.

Reply:
Updated so `sfixtor` is renamed to `stor_fix` and `rtosfix` is renamed to `rtos_fix`.